.ci/windows/versions.json

```diff
@@ -1,4 +1,4 @@
 {
-  "erlang": "26.2.2",
+  "erlang": "26.2.3",
```

Comment:
Given that `26.2.3` has a known [critical issue on aarch64](https://github.com/erlang/otp/pull/8248), should we stick to `26.2.2` for now? any reasons to use `26.2.3` even if we do not run CI on aarch64?

Reply:
I just try to stick to the latest. We don't have to worry about that `aarch64` issue with GitHub actions.